docs/concepts/pdp/configuration.mdx

```diff
@@ -315,6 +329,70 @@ Default: `False`
 
 Enable FactDB for this PDP. For more information, see the [FactDB documentation](/concepts/pdp/factdb).
 
+#### PDP_CACHE_STORE
```

Comment:
I think it's PDP_CACHE__STORE ( with double `_` because it's nested )

Reply:
Nope, that was changed